core/src/main/scala/scala/pickling/spi/PicklerRegistry.scala

```diff
@@ -98,5 +98,7 @@ trait PicklerRegistry {
     *                   this type.
     */
   def registerPicklerUnpicklerGenerator[T](typeConstructorKey: String, generator: FastTypeTag[_] => (Pickler[T] with Unpickler[T])): Unit
-  // TODO - Some kind of clean or inspect what we have?
+
+  private[pickling] def clearRegisteredPicklerUnpicklerFor[T: FastTypeTag]: Unit
```

Comment:
nit: Can you add some quick docs explaining this?  We understand, but someone new to the codebase will be curious.